docs: update docs inline latest syntax changes (#4686)
confluentinc/ksql: docs/developer-guide/create-a-table.md

### Create a Table with Selected Columns

The following example creates a table that has four columns from the
`users` topic: `registertime`, `userid`, `gender`, and `regionid`. 

> The `userid` column is the primary key of the table. This means it is loaded from the Kafka message

Reviewer: ```suggestion
The `userid` column is the primary key of the table. This means that it's loaded from the {{ site.ak }} message
```

> key. Primary key columns can not be `NULL`.

Reviewer: ```suggestion
key. Primary key columns can't be `NULL`.
```
